docs/whats_new.rst

```diff
@@ -5,23 +5,40 @@
 
 .. _whats_new:
 
-Changelog
-============
+The NextCord Update Changelog
```

Comment:
There already in the Nextcord docs, is this really needed?

Reply:
wdym?

Reply:
its just `Changelog`, why does it need to specify its nextcord